docs/generalqa.md

```diff
@@ -0,0 +1,39 @@
+# General Q&A
+
+This area of Archipelago documentation is reserved for general questions and answers for commonly encountered issues pertaining to Archipelago configuration settings.
+
+To contribute to this section, please review our [Code of Conduct](CODE_OF_CONDUCT.md), and after that please follow [this set of guidelines](docs/giveortake.md) to help you get started.
+
+---
+
+## Twig Modules Configuration
+**Q:** When attempting to save a Twig template for a Metadata Display, I receive an error message related to an `Unknown "bamboo_load_entity" function`.
+
+![BambooTwigError](../imgs/generalqa/BambooTwigError.jpg)
+
+**A:** You need to enable the necessary Twig modules.
+
+1. Naviate to: yoursite/admin/modules
```

Comment:
```suggestion
1. Navigate to: `yoursite/admin/modules`
```
Tiny typo here. Also we need to make sure we always suggest links to their sites in the same way. I like the yoursite. I also recommended adding single quotes to links / terms/concepts so they look different